lib/src/utils/nutripatrol_types.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:json_annotation/json_annotation.dart';
+
+enum NutripatrolSource {
```

Comment:
I would
* extend `OffTagged`
* put each `enum` in a specific file
* add "Nutripatrol" and remove "Enum" in your comments

But that's not mandatory.